Added missing translations (Swedish)

devmaximilian opened the pull request:
Updated Swedish translations to reflect new features. Relates to #65. Let me know if there is anything I should change!

Schlaubischlump commented:
Thanks for the updated translation ! 

I just got two small questions. I checked the translation for "Zoom In" and "Zoom Out" in the maps app and it says "Zooma in" and "Zooma ut". Should I leave the english way or is the maps translation more correct ? 

The translation for "IMAGE" is used in the context of "disk image". I don't know if this changes the Swedish translation. In german the word "Bild" is most often used when referring to pictures, thats why I thought it might be the same way in Swedish.

devmaximilian commented:
Thank you for the feedback! 

The zoom-related translations were a miss on my part, they are indeed correct in Apple Maps. I've updated the translations to reflect this.

"Disk image" is "skivavbild" in Swedish and I have adjusted the translations to reflect this properly. 

I translated "DeveloperDiskImage" in contexts where text is not directly referring to the name of a file (.dmg, .dmg.signature). For this translation, I used the word "utvecklarskivavbild" (triple-union word) instead of "skivavbild för utvecklare" (which is what "developer disk image" translates to). However, it could also be referred to as just "skivavbild", which would simplify things.

Let me know if I should change it either way!

Schlaubischlump commented:
Thanks for the explanation! It looks good, I'll merge it.
